superset/assets/javascripts/SqlLab/components/SqlEditor.jsx

```diff
@@ -226,6 +229,32 @@ class SqlEditor extends React.PureComponent {
   render() {
     const height = this.sqlEditorHeight();
     const defaultNorthHeight = this.props.queryEditor.height || 200;
+    const SQLLAB_HOTKEYS = [
```

Comment:
nit: Can we put this at the top of the file or even in config file for adding new hotkeys more easily. I'm like the idea of a having this separate and we import it

Reply:
Ahhh true, the `func` param, i just don't like the bulky render view, but if there isn't a way to clean it up we can just go with it.

Reply:
moved to `this.getHotkeyConfig()`